perf_test/sparse/KokkosSparse_sptrsv_supernode.cpp

```diff
@@ -0,0 +1,445 @@
+/*
+//@HEADER
+// ************************************************************************
+//
+//               KokkosKernels 0.9: Linear Algebra and Graph Kernels
```

Comment:
@iyamazaki The header info will need to be updated with the new 3.0 copyright info, you can look at another file and just copy+replace the header here.